packages/box/src/dfuse-receiver.ts

```diff
@@ -83,6 +83,7 @@ async function webSocketFactory(
 
 export default class DfuseReceiver {
     storage: Storage;
+    stream: Stream | null = null;
```

Comment:
question: why not `undefined`? it plays better with typescript

Reply:
I got in the habit of using null since json and graphql don't have undefined